### PR TITLE
Fixes for sorting by proximity of user

### PR DESCRIPTION
**Pull request by MiletaStanisic**
https://app.asana.com/0/1137432207089210/1131076422192598
https://app.asana.com/0/1137432207089210/1151314464045457

### CODE REVIEW

**Comment by MiletaStanisic** [comment]
https://app.asana.com/0/1137432207089210/1131076422192598
https://app.asana.com/0/1137432207089210/1151314464045457

File: src/widget/js/app.js [hunk 1]
```diff
@@ -229,18 +227,49 @@ window.app = {
...
+              return -1;
+            }
+            if (aDistance > bDistance) {
+              return 1;
+            }
+            return 0;
+          });
+          window.lazyload();
+          window.listView.sorting(window.app.state.places);
+          window.lazyload();
```
Comment (denza) [inline src/widget/js/app.js]: Why calling lazyload 2x?

File: src/widget/js/list.js [hunk 2]
```diff
@@ -26,105 +26,85 @@ window.listView = {
...
-                    noDistanceData.push(place);
-                }
-            });
-            feetPlaces = feetPlaces.sort(window.PlacesSort[window.app.state.sortBy]);
-            milesPlaces = milesPlaces.sort(window.PlacesSort[window.app.state.sortBy]);
-            sortPlaces = feetPlaces.concat(milesPlaces).concat(noDistanceData);
-        } else {
-            sortPlaces = places.sort(window.PlacesSort[window.app.state.sortBy]);
+        window.listView.sorting(places);
+        window.lazyload();
```
Comment (denza) [inline src/widget/js/list.js]: Calling lazyload 2x

File: src/widget/js/app.js [hunk 3]
```diff
@@ -229,18 +227,48 @@ window.app = {
...
+            }
+
+            if (aDistance < bDistance) {
+              return -1;
+            }
+            if (aDistance > bDistance) {
+              return 1;
+            }
+            return 0;
+          });
```
Comment (denza) [inline src/widget/js/app.js]: Extract to a method

**Comment by denza** [comment]
@najiatallah Looks good

**Comment by najiatallah** [comment]
Approved

**Pull request merged**